src/Containers/Impl/Docker/LocalDockerService.cs

```diff
@@ -138,8 +141,19 @@ public abstract class LocalDockerService : IDockerService {
             string output = await process.StandardOutput.ReadToEndAsync();
             string error = await process.StandardError.ReadToEndAsync();
             if (!string.IsNullOrEmpty(error)) {
+                try {
+                    _outputLogWriter?.Write(MessageCategory.Error, error);
+                } catch(Exception ex) when (!ex.IsCriticalException()) {
```

Comment:
`FileLogWriter` handles all IO-related exceptions exceptions by itself (see `WriteToFileAsync`), so I think `OutputWindowLogWriter` should do the same.